apis/nucleus/src/components/listbox/ListBoxInline.jsx

```diff
@@ -137,7 +146,7 @@ export function ListBoxInline({ app, fieldName, stateName = '$', options = {} })
         <Grid item>
           {showTitle && (
             <Typography variant="h6" noWrap>
-              {layout.title || fieldName}
+              {!isMaster ? layout.title || fieldName : layout.title || layout.qListObject.qDimensionInfo.qFallbackTitle}
```

Comment:
when not a master item `layout.qListObject.qDimensionInfo.qFallbackTitle` should be the same as `fieldName`
```suggestion
              {layout.title || layout.qListObject.qDimensionInfo.qFallbackTitle}
```